core/modules/search/search.module

```diff
@@ -1059,13 +1062,15 @@ function search_form($form, &$form_state, $action = '', $keys = '', $module = NU
  * @see search_box_form_submit()
  */
 function search_box($form, &$form_state, $form_id) {
+  // This field is not required so that the user is taken to the search page
```

Comment:
The language here is a bit uncomfortable. How about "The `#required` property is not set here, so that ..."

Reply:
Agreed. Altered comment needs rewrap.